client/executor/wasmtime/src/imports.rs

```diff
@@ -67,7 +67,7 @@ where
 				log::debug!("Missing import: '{}' {:?}", name, func_ty);
 				linker
 					.func_new("env", &name, func_ty.clone(), move |_, _, _| {
-						Err(Trap::new(error.clone()))
+					    Err(anyhow::anyhow!("{error}"))
```

Comment:
Why isn't that just a `Box<Error>`?

Reply:
https://docs.rs/anyhow/1.0.68/anyhow/struct.Error.html#impl-From%3CE%3E-for-Error should be possible.

Reply:
The `error` here is a `String`, and this needs to return `anyhow::Error` since that's what `wasmtime` requires. But I guess I could use`anyhow::Error::msg(error.clone())` instead of `anyhow!`.

Reply:
The macro wasn't the problem :D But I still think that `Box::<dyn StdError>::from(error.to_string()).into()` should  work :D

Reply:
That is... quite the Rube Goldberg contraption. (:

From what I can see that doesn't work though as `Box<E>` only implements `Error` when `E` is `Sized`, and `dyn StdError` is not `Sized`.